react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class NewsletterForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: '' };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  render() {
+    return (
+      <form>
+        <label htmlFor="email">Email:</label>
+        <input onChange={this.handleChange} type="email" value={this.state.value}/>
+        <button onClick={this.handleSubmit} type="submit" value="submit">Sign Up</button>
```

Comment:
You want to listen for submit events on the `<form>` instead of click events on the `<button>`. That way, the form can be submitted with the Enter key

Reply:
Thanks for the feedback. Made the changes and re-tested. Appears to be working with the enter key and clicking the button now!

![Kapture 2021-10-21 at 16 33 50](https://user-images.githubusercontent.com/74282257/138370825-212f7e4e-9d17-4904-9f4e-df7c8c9193af.gif)